health-check/index.js

```diff
@@ -6,26 +6,71 @@
  */
 
 const { v1: uuidv1 } = require('uuid');
-const mongodb = require('./services/mongodb');
-const kafka = require('./services/kafka');
 const httpService = require('./services/httpService');
 
-async function healthCheckHandler(config) {
+async function healthCheckHandler(config,currentServiceName = '') {
 	const checks = [];
-	
-	if (config.checks.mongodb?.enabled) {
-		const healthy = await mongodb.check(config.checks.mongodb.url);
-		checks.push(serviceResult('MongoDB', healthy));
+
```

Comment:
@MallanagoudaB add descriptive comments for each if condition for others to understand.